schedule/yast/maintenance/qam-yast_self_update_sle15sp3_dev.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: qam-yast_self_update
+description: installation using self_update as boot parameter
+vars:
+  YUI_REST_API: 1
+schedule:
+  license_agreement:
+    - installation/validate_self_update
+    - installation/licensing/accept_license
+  add_on_product:
+    - installation/add_on_product/add_maintenance_repos
+    - installation/addon_products_sle
```

Comment:
I think line was not needed, could you please check it? (if so, apply also in next test suite)